Expo1900/Expo1900/Controller/ExpoMainViewController.swift

```diff
@@ -0,0 +1,189 @@
+        stackView.axis = .vertical
+        stackView.alignment = .center
+        stackView.distribution = .equalSpacing
+        stackView.spacing = DetailSetUp.stackViewSpacing
+        return stackView
+    }()
+    
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.preferredFont(forTextStyle: .title2)
+        label.numberOfLines = DetailSetUp.labelNumberOfLines
+        label.textAlignment = .center
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    private let descriptionLabel: UILabel = {
+        let label = UILabel()
+        label.numberOfLines = DetailSetUp.labelNumberOfLines
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    private let subStackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .horizontal
+        stackView.alignment = .center
+        stackView.distribution = .equalCentering
+        return stackView
+    }()
+    
+    private let nextViewButton: UIButton = {
+        let button = UIButton()
+        button.setTitle(DetailSetUp.buttonTitle, for: .normal)
+        button.setTitleColor(UIColor.blue, for: .normal)
+        button.setContentCompressionResistancePriority(DetailSetUp.buttonCompressionResistancePriority, for: .horizontal)
+        button.addTarget(nil, action: #selector(tappedNextViewButtonEvent), for: .touchUpInside)
+        return button
+    }()
+    
+    //MARK: - View Life Cycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.navigationItem.title = DetailSetUp.title
+        
+        setMainView()
+        setLabelText()
+        addItemStackView()
+        setViewConstraints()
+    }
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        self.navigationController?.isNavigationBarHidden = true
+    }
+    
+    //MARK: - Setting View Methods
+    
+    private func setMainView() {
+        self.view.backgroundColor = .systemBackground
+        self.view.addSubview(mainScrollView)
+        self.mainScrollView.addSubview(mainStackView)
+    }
+    
+    private func setLabelText() {
+        guard let expoInformation = fetchExpoInformation(from: Asset.expoInformation) else { return }
+        expoData = expoInformation.toDomain()
+        titleLabel.text = expoData?.title
+        descriptionLabel.text = expoData?.description
+    }
+    
+    private func addItemStackView() {
+        let mainStackViewItemsArray = [
+            titleLabel,
+            createImageView(imageName: Asset.posterImage),
+            createLabel(expoData?.visitors),
+            createLabel(expoData?.location),
+            createLabel(expoData?.duration),
+            descriptionLabel,
+            subStackView]
```

Comment:
마지막 단이 더 잘 구분될 수 있게 줄바꿈 해주는 것도 괜찮아 보입니다~!
```suggestion
            subStackView
        ]
```